code/modules/hydroponics/grown/potato.dm

```diff
@@ -30,7 +30,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/potato/wedges
```

Comment:
If that were the case, sweet potatos would have the same problem, which they don't